.php-cs-fixer.php

```diff
@@ -157,7 +157,7 @@
             'case' => 'snake_case',
         ],
         'php_unit_set_up_tear_down_visibility' => true,
-        'php_unit_strict' => true,
+        'php_unit_strict' => false,
```

Comment:
unfortunately this setting forces to compare objects by reference which we do not want

Reply:
is it possible to ignore it in 1 place instead of disable at all?